rigid_opt/sdf_gradient_field.py

```diff
@@ -0,0 +1,42 @@
+#  ================================================================
+#  Created by Fei Shan on 02/01/19.
+#  Calculate sdf gradient w.r.t. transformation vector
+#  ================================================================
+
+# common libs
+import numpy as np
+
+# local
+from rigid_opt.transformation import twist_vector_to_matrix
+
+
+class GradientField:
```

Comment:
Why do we have a whole separate class for this? It looks like this could be simplified to a single method, i.e. ```calculate_gradient_field(live_field, twist, array_offset, voxel_size=0.004)```

Reply:
I guess it could. I did it this way thinking it might be better for further extension, i.e. adding more functions.

Reply:
Please, simplify for now.